cmd/skaffold/app/cmd/build.go

```diff
@@ -41,6 +41,11 @@ var (
 func NewCmdBuild() *cobra.Command {
 	return NewCmd("build").
 		WithDescription("Build the artifacts").
+		WithLongDescription("Build, test and tag the artifacts").
+		WithExample("Build all the artifacts", "build").
+		WithExample("Build artifacts with a profile activated", "build -p <profile>").
+		WithExample("Build artifacts whose image name contain <db>", "build -b <db>").
+		WithExample("Quietly build artifacts and output the image names as json", "build -q -o '{{json .}}'").
```

Comment:
you don't need the `-o '{{json .}}'`, by default -q spits out the json on stdout...

Reply:
How about ` skaffold build -q > build_result.json`
+ one more example candidate: `skaffold build -q > skaffold deploy`